modules/arg.py

```diff
@@ -270,6 +271,28 @@ async def media(self, interaction=Interaction, *, url):
             await interaction.followup.send(f"Failed. Unknown response code: {resp.status}. Please contact the bot's creator kthx.")
         return
 
+    @commands.slash_command(
+        name="time", description="Posts how much time is left until the next tweet."
+    )
+    async def time(self, interaction=Interaction):
+        todays_date = datetime.datetime.now()
+        todays_tweet_post_date = todays_date.replace(hour=23, minute=00)
+        tomorrows_tweet_post_date = todays_tweet_post_date + datetime.timedelta(days=1)
+        first_tweet_date = datetime.datetime(2022,5,7)
+
+        if todays_date.time() < datetime.time(23,00):
+            unix_timestamp = datetime.datetime.timestamp(todays_tweet_post_date)
+        else:
+            unix_timestamp = datetime.datetime.timestamp(tomorrows_tweet_post_date)
+
+        if (todays_date - first_tweet_date).days % 2 == 0:
+            if(todays_date.time() < datetime.time(23,00)):
+                tweet_sender = "Aine"
```

Comment:
If there's ever going to be more people, then we would have to edit the code again.

Maybe it would be better to store the values in the environment?